notebooks/post_DL3_analysis.ipynb

```diff
@@ -0,0 +1,2655 @@
+{
```

Comment:
<p>You should probably integrate only over the valid fit range here. If the 23-89 GeV range is not safe for spectrum extraction it is probably not safe for the light curve. Especially since we extrapolate the spectrum in this range.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cta-observatory/cta-lstchain/pull/955/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='51'/>

Reply:
Agreed. Will fix it